blocks/lead-art-block/_index.scss

```diff
@@ -15,6 +15,17 @@
 		@include scss.block-components("lead-art-close-icon");
 		@include scss.block-properties("lead-art-close-icon");
 	}
+
+	&__button--exit-full-screen {
+		@include scss.block-components("lead-art-button-exit-full-screen");
+		@include scss.block-properties("lead-art-button-exit-full-screen");
+	}
+
+	&__icon--exit-full-screen {
+		@include scss.block-components("lead-art-icon-exit-full-screen");
+		@include scss.block-properties("lead-art-icon-exit-full-screen");
+	}
```

Comment:
You can use
``` 
"components": {
  "button": {}
  "icon": {}
}
```
to style buttons and icons within a block. This would be preferable to creating new tokens accessing the same elements.

Reply:
Comment attended on refactor.